package.json

```diff
@@ -18,10 +18,16 @@
   },
   "homepage": "https://github.com/awaisanwar544/portfolio#readme",
   "devDependencies": {
-    "hint": "^6.1.9",
+    "eslint": "^8.7.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.25.4",
+    "hint": "^6.1.3",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^21.0.0",
     "stylelint-csstree-validator": "^1.9.0",
     "stylelint-scss": "^3.21.0"
+  },
+  "dependencies": {
+    "eslint-config-airbnb": "^19.0.4"
```

Comment:
- [ ] [OPTIONAL] please kindly remove this line on your package.json. You already have `eslint-config-airbnb` as a `devDependency` so this is extraneous.